extensions/cornerstone/src/getToolbarModule.tsx

```diff
@@ -20,9 +45,32 @@ export default function getToolbarModule({ commandsManager, servicesManager }: w
   return [
     // functions/helpers to be used by the toolbar buttons to decide if they should
     // enabled or not
+    {
+      name: 'evaluate.not.sm',
```

Comment:
This is quite ugly - how about
evaluate.viewport.supported
evaluate({ viewportId, button }) => {
      const viewport = ...
      if( button.unsupportedTypes.includes(viewport?.type) ) return getDisabledState().

Then, you can get rid of tools not supported by video viewport as well as the evaluate actions.

Reply:
I thought i had it like this, hmm weird